src/dra/692E8.c

```diff
@@ -304,7 +304,753 @@ TeleportCheck GetTeleportToOtherCastle(void) {
+                                g_Player.D_80072F00[4] = 0xC;
+                                g_Player.D_80072F00[15] = 0xC;
+                                func_8010E168(1, 0xC);
+                            }
+                            continue;
+                        case 1:
+                            if (!(g_Player.unk0C & 0x8080)) {
+                                g_Player.D_80072F00[4] = 0xC;
+                                g_Player.D_80072F00[15] = 0xC;
+                                func_8010E168(1, 0xC);
+                            }
+                            continue;
+                        case 2:
+                            PLAYER.palette = 0x8100;
+                            continue;
+                        case 3:
+                            PLAYER.palette = 0x8100;
+                            if (!(g_Player.unk0C & 0xC080)) {
+                                g_Player.D_80072F00[4] = 0xC;
+                                g_Player.D_80072F00[15] = 0xC;
+                                func_8010E168(1, 0xC);
+                            }
+                            continue;
+                        case 4:
+                            draw->enableColorBlend = 0;
+                            continue;
+                        case 6:
+                            if ((PLAYER.step == 3) &&
+                                (PLAYER.ext.player.unkAC != 0x1C)) {
+                                func_8010DA48(0x1C);
+                                g_Player.unk44 &= 0xFFEF;
+                            }
+                            continue;
+                        case 13:
+                            func_8010E168(1, 0x10);
+                            continue;
+                        case 15:
+                            func_8010E0B8();
+                            continue;
+                        case 11:
+                            func_801092E8(0);
+                            continue;
+                        }
+                    }
+                }
+            }
+            if (D_800ACDFC != 0) {
+                D_800ACDFC--;
+            }
+            g_Player.padHeld = g_Player.padPressed;
+            if (g_Player.D_80072EFC != 0) {
+                g_Player.D_80072EFC--;
+#ifdef VERSION_US
+                if (g_Player.D_80072EFC == 0) {
+                    D_80137FBC = 1;
+                }
+#endif
+                g_Player.padPressed = g_Player.D_80072EF4 & 0xFFFF;
+                switch (g_Player.D_80072EF4 >> 0x10) { /* switch 6; irregular */
+                case 1:                                /* switch 6 */
+                    if (PLAYER.step != 0x30) {
+                        func_8010E168(1, 4);
+                        SetPlayerStep(0x30);
+                        *D_80097400 = 1;
+                    }
+                    break;
+                case 2: /* switch 6 */
+                    func_8010E168(1, 4);
+                    if (g_Player.unk0C & 0x01000000) {
```

Comment:
PLAYER_STATUS_AXEARMOR
there's various other spots where some of the could be documented